README.md

```diff
@@ -1 +1,40 @@
-# Hello-Microverse-Reviewable
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello World
+
+> A basic Hello Microverse Project with HTML and CSS. It has Linters Implemented and ignored redundant files using .gitignore
+
+![screenshot]()
+
+
+## Built With
+
+- VSCODE
+- Github
+- Git
+
+## Getting Started
+
+
```

Comment:
## Changes Required

⚠️ Add some information under this section so people know how to use your repo! You could add a link to a deployment, or instructions on how to clone your repo and open the files on your computer.